apps/tangle-dapp/app/restake/unstake/types.ts

```diff
@@ -0,0 +1,6 @@
+export type UnstakeRequestTableData = {
+  uid: string;
+  amount: number;
+  assetSymbol: string;
```

Comment:
The asset data is fetched on-chain and is dynamic, so I opted for a general type for these properties. 

We should avoid hardcoding whenever possible, as it would require updates if the backend changes.